src/helpers/audio.helper.ts

```diff
@@ -0,0 +1,66 @@
+import { ICall, StreamMediaType, MediaEvent } from '@/types/rtc'
+
+type ICallKey = keyof ICall
+const CALL_KEYS_TO_INCLUDE: Array<ICallKey> = [
+    'roomId',
+    '_audioMuted',
+    '_cancel_reason',
+    '_contact',
+    'direction',
+    '_end_time',
+    '_eventsCount',
+    '_from_tag',
+    '_id',
+    '_is_canceled',
+    '_is_confirmed',
+    '_late_sdp',
+    '_localHold',
+    '_videoMuted',
+    'status',
+    'start_time',
+    '_remote_identity',
+    'audioTag',
+    //'audioQuality',
+    'isOnHold',
+    //'originalStream',
+    'localMuted'
+]
+
+export function simplifyCallObject (call: ICall): { [key: string]: any } {
+    //const simplified: { [key: string]: ICall[ICallKey] } = {}
+    const simplified: { [key: string]: any } = {} as ICall
```

Comment:
You can do better. TS has two interesting generic functions:
- Omit
- Pick

Both of them allows us to manipulate the object type/interface working with their keys, Omit allows us to create new type by providing list of keys to ignore, Pick does the opposite. As we already has the list of keys we want to include, suggest to do the following:
```ts
type Writeable<T> = { -readonly [P in keyof T]: T[P] }

export type ICallSimplified = Writeable<Pick<ICall, typeof CALL_KEYS_TO_INCLUDE[number]>>

export function simplifyCallObject (call: ICall): ICallSimplified {
    const simplified: Partial<ICallSimplified> = {}

    CALL_KEYS_TO_INCLUDE.forEach((key) => {
        if (call[key] !== undefined) {
            simplified[key] = call[key]
        }
    })

    simplified.localHold = call._localHold

    return simplified as ICallSimplified
}
```

Just move the Writeable to some generic types